### PR TITLE
Unit and integration tests.

### PR DESCRIPTION
**Pull request by mkryva**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -62,7 +62,12 @@ lint: ## check style with flake8
 	flake8 twindb_infrastructure tests
 
 test: bootstrap ## run tests quickly with the default Python
-	py.test --flakes --full-trace --verbose --cache-clear tests/
+	py.test --flakes --full-trace --verbose --cache-clear tests/unit
+
+test-integration: bootstrap
+	pip show twindb-infrastructure || pip install -e
```
Comment (akuzminsky) [inline Makefile]: Move all preparation to the `bootstrap` target

File: tests/integration/providers/test_twindb_aws.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                "MountPoint": "/var/log/mysql"
+            }
+        ],
+        "InstanceType": "t2.micro",
+        "KeyName": "tester",
+        "SecurityGroupIds": ["sg-de8a66a5"],
+        "SubnetId": "subnet-f0190aa8",
+        "RootVolumeSize": 200,
+        "AvailabilityZone": "us-west-2c",
+        "UserName": "mkryva",
```
Comment (akuzminsky) [inline tests/integration/providers/test_twindb_aws.py]: it should be `centos` or `ubuntu` depending on AMI you use

File: tests/integration/providers/test_twindb_aws.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        "Name": "integraion-test-01",
+        "Region": "us-west-2"
+    }
+
+    try:
+        instance_id = launch_ec2_instance(instance_profile, region=instance_profile['Region'])
```
Comment (akuzminsky) [inline tests/integration/providers/test_twindb_aws.py]: it's not enough just to launch the instance.
You should make sure it's launched and entered state `running` after some time (with timeout)

Comment (mkryva) [inline tests/integration/providers/test_twindb_aws.py]: But, this checking in launch_ec2_instance implementation, doesn't it ?

**Pull request merged**